common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -429,15 +405,10 @@ nav.sequence-bottom {
   }
 }
 
-.xmodule_VerticalModule div.vert-mod > div ul.sequence-nav-buttons {
-  list-style: none !important;
-}
-
-.xmodule_SequenceModule nav.sequence-bottom ul li.next a,
-.xmodule_SequenceModule nav.sequence-bottom ul li.prev a {
+#seq_content {
 
-/*  &:focus {
-    outline: 0;
+  &:focus,
+  &:active {
+    outline: none;
```

Comment:
Any chance we can make this a Sass extend that can be consistently applied in all of the cases where we don't want an outline? Conversely, it'd be great to have an outline extend we use when we do want consistent show visual focus.